README.md

```diff
@@ -13,24 +13,24 @@
 ├── .gitignore
 |
 ├── data
-│   ├── demofile.json
-│   └── testfile.json
+│   ├── demofile.json
+│   └── testfile.json
```

Comment:
Similarly, does the testfile need to appear in the README path? Anyone attempting to use/modify this program would probably only need to know about it with regard to testing.